packages/redux-api-auth-middleware/src/store.js

```diff
@@ -1,21 +1,39 @@
 // @flow
+import { RSAA } from 'redux-api-middleware';
```

Comment:
not sure if the filename choice is the best here - as there is no store in this file, it's more of a 🦆

Reply:
What would you recommend? Maybe just name it `redux.js`?

Reply:
It's not good either. I mean, it's only a filename, we shouldn't think about that much. The common name for that type of file is just `reducer.js`.